LayoutTests/platform/mac-ventura/TestExpectations

```diff
@@ -1,3 +1,4 @@
 # Failing after OS migration rdar://112624778 (Migrate macOS Sonoma test expectations to OpenSource, add expectation files to Down-Levels (259373))
 
+webkit.org/b/266630 imported/w3c/web-platform-tests/trusted-types/trusted-types-createHTMLDocument.html [ Pass Failure ]
```

Comment:
Move this to the top-level TestExpectations file and link it to https://bugs.webkit.org/show_bug.cgi?id=272196